miqa/core/tests/pyppeteer/test_scan_upload.py

```diff
@@ -63,6 +63,6 @@ async def test_upload_scans(
     await scans[0].click()
 
     assert await page.waitForXPath(
-        '//div[contains(@class, "col")][contains(., "Demo Project")]'
-        '/following::div[contains(@class, "col")][contains(., "Test Experiment")]'
+        '//div[contains(@class, "d-flex")][contains(., "Demo Project")]'
+        '/following::div/div[contains(., "Test Experiment")]'
```

Comment:
Same here; rather than depending on a specific nesting of the DOM, we should just add a special class to the element where this text is expected.

Reply:
Changed in [0204ae3](https://github.com/OpenImaging/miqa/pull/563/commits/0204ae3d19d755b3fe8e7acbd71355c81b7a63be)